extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -489,6 +508,26 @@ export class ConsentUI {
     return srcUrl.origin != containerUrl.origin;
   }
 
+  /**
+   * Retrieve additional sandbox restrictions to be removed from the iframe.
+   * @return {Array<string>}
+   */
+  getAdditionalSandboxAttributes_() {
+    return (this.config_['sandbox'] || '')
```

Comment:
Yes, it is about adding further values. I did not want to touch the logic related to the already existing values (`allow-scripts`, `allow-popups` and `allow-same-origin`) to prevent breaking currently working setups. Therefore, I decided to just allow adding further sandbox values to the pre-defined list.

Reply:
sounds good. will merge